Source/ZoomNet/Json/ZoomNetJsonSerializerContext.cs

```diff
@@ -18,7 +19,7 @@ namespace ZoomNet.Json
 	[JsonSerializable(typeof(ZoomNet.Models.AuthenticationSettings))]
 	[JsonSerializable(typeof(ZoomNet.Models.AuthenticationType))]
 	[JsonSerializable(typeof(ZoomNet.Models.BatchRegistrant))]
-	[JsonSerializable(typeof(ZoomNet.Models.BatchRegistrant[]))]
+	[JsonSerializable(typeof(IEnumerable<ZoomNet.Models.BatchRegistrant>))]
```

Comment:
Hello @Jericho, I'm unable to find this line in both the main and development branches, which is causing an exception. Is there any way to resolve this issue? It seems that the same problem is also occurring with the released NuGet package.

![image](https://user-images.githubusercontent.com/29385369/228687926-45f6dd64-2f6e-4954-86d6-cd565aa65979.png)

Reply:
This is caused by the fact that registrants are not converted to an array prior to serialization in PerformBatchRegistrationAsync. This code:
```
{ "registrants", registrants },
```

should be:
```
{ "registrants", registrants.ToArray() },
```

I'll fix it.

Reply:
Thanks a lot mate. It's working perfectly now.